MobiFlight/InputConfig/ButtonInputConfig.cs

```diff
@@ -69,20 +77,36 @@ public void WriteXml(System.Xml.XmlWriter writer)
             writer.WriteStartElement("onRelease");
             if (onRelease != null) onRelease.WriteXml(writer);
             writer.WriteEndElement();
+

```

Comment:
an option could be that we don't write the entire node if it is not set. Also true for the other nodes, would reduce the XML output.

In this case, we could even open the config in an old MF version in case no onReleaseLong is defined.

Reply:
Done